src/iotjs_binding.cpp

```diff
@@ -96,6 +101,21 @@ JObject::JObject(const String& v) {
 }
 
 
+JObject::JObject(uint32_t len, const char* data) {
+  _obj_val = jerry_create_array(len);
+
+  if (data != NULL) {
+    for (uint32_t i = 0; i < len; i++) {
+      jerry_value_t val = jerry_create_number(double(data[i]));
```

Comment:
Yes. It works in my environment (gcc-4.8 for x86_64-linux and arm-linux). I've seen this style since sevral years ago. Probably it is a new syntax from new version C or C++. However, you're right. for portability, I will change to normal style. Thank you for your review.